src/rockstor/storageadmin/views/pool.py

```diff
@@ -51,6 +53,54 @@ def _validate_disk(d, request):
             handle_exception(Exception(e_msg), request)
 
     @staticmethod
+    def _get_disk_names(disks, request):
```

Comment:
It can be removed if you preferred this. I used it during development and though that it might come in again for future development so left it in place, but it can always be recreated of course.